tools/packaging/build/agent-enclave-bundle/Dockerfile

```diff
@@ -1,5 +1,12 @@
 FROM rust:1.63-bullseye as builder
 
+RUN git clone --depth 1 https://github.com/inclavare-containers/rats-tls.git && \
```

Comment:
It is recommended to use an env to represent the commit id of rats-tls, which is more conducive to later maintenance (In line 5 and line 66)，such as:
```
ENV RATS_TLS_REV 5de6fc308f2a18f4105429451c85e6173b42ab9d
git checkout $RATS_TLS_REV (In line 5 and line 66)
```

Reply:
Why are we building at the commit ID where RA-TLS is updated? i.e., because that commit is in main why don't we just build at the current head commit? We may miss future bug fixes if we peg to that commit.

Also per @mythi 's earlier comment, if we do not clone the commit history we will not clone this commit.

Reply:
Disregard my question about building against a specific commit. I see now in https://github.com/confidential-containers/enclave-cc/issues/76 where @mythi recommended doing that to avoid breaking our build if main is volatile. We will treat this commit ID like a stable tag.

My other comment about shallow clone not getting this commit is still relevant, though.

Reply:
Sure, option `--depth 1` has been removed.

Reply:
One thing I was not very clear in [my comment](https://github.com/confidential-containers/enclave-cc/issues/76#issuecomment-1350483311) was related to "let's not do it twice". I wanted to say that instead of installing rats-tls/SGXSDK twice, we should move away from `rust:1.63-bullseye` and just install rust ourselves.

Reply:
> we should move away from `rust:1.63-bullseye` and just install rust ourselves.

I'll submit a PR about this.

Reply:
see #80